dockstore-webservice/src/main/java/io/dockstore/webservice/resources/HostedWorkflowResource.java

```diff
@@ -186,4 +188,14 @@ protected boolean checkValidVersion(Set<SourceFile> sourceFiles, Workflow entry)
         }
         return false;
     }
+
+    @Override
+    protected String checkType(String descriptorType) {
+        if (!Objects.equals(descriptorType.toLowerCase(), DescriptorLanguage.CWL_STRING)
```

Comment:
It would be nice if you could instead iterate through all the DescriptorLanguage enum values instead of checking against a hard-coded list of languages; that way if a new workflow language is added, you don't have to update this code.

E.g., call `DescriptorLanguages.values()` and go from there.

Unfortunately DescriptorLanguage doesn't have the lower case descriptor type, so it might be more hassle than it's worth. Up to you...